regression/contracts/quantifiers-forall-ensures-enforce/test.desc

```diff
@@ -3,7 +3,10 @@ main.c
 --enforce-all-contracts
 ^EXIT=0$
 ^SIGNAL=0$
-^\[1\] assertion: SUCCESS$
+^\[postcondition.\d+\] file main.c line \d+ Check ensures clause: SUCCESS$
+^\[f1.\d+\] line \d+ Check that i is assignable: SUCCESS$
+^\[f1.\d+\] line \d+ Check that i is assignable: SUCCESS$
+^\[f1.\d+\] line \d+ Check that arr\[\(signed long int\)i\] is assignable: SUCCESS$
```

Comment:
Ah on Windows this is `signed long long int`:

```
[f1.2] line 18 Check that arr[(signed long long int)i] is assignable: SUCCESS
```

I would just make this:

```
^\[f1.\d+\] line \d+ Check that arr\[\(.*\)i\] is assignable: SUCCESS$
```

Reply:
Fixed it!